packaging/deb-systemd/debian/control

```diff
@@ -9,7 +9,7 @@ Vcs-Git: https://github.com/mackerelio/mackerel-agent.git
 Homepage: https://mackerel.io
 
 Package: mackerel-agent
-Architecture: amd64
+Architecture: any
```

Comment:
ref: https://www.debian.org/doc/debian-policy/ch-controlfields.html#architecture

When `any`, produced binary package will have specific architecture (amd64, mips, etc…), which is desired behavior.